crypto/schemes.go

```diff
@@ -197,7 +194,46 @@ func NewPedersenBLSUnchainedSwapped() (cs *Scheme) {
 		ThresholdScheme: ThresholdScheme,
 		AuthScheme:      AuthScheme,
 		DKGAuthScheme:   DKGAuthScheme,
-		Pairing:         Pairing,
+		IdentityHash:    IdentityHashFunc,
+		DigestBeacon:    DigestFunc,
+	}
+}
+
+// SigsOnG1ID is the scheme id used to set unchained randomness on beacons with signatures on G1 that are
+// compliant with the hash to curve RFC.
+const SigsOnG1ID = "bls-unchained-g1-rfc"
+
+// NewPedersenBLSUnchainedG1 instantiate a scheme of type "bls-unchained-on-g1" which is also unchained, only
+// hashing the round number as the message being signed in beacons. This scheme is also compatible with
+// "timelock encryption" as done by tlock.
+// This schemes has the group public key on G2, so 96 bytes, and the beacon signatures on G1, so 48 bytes.
+// This means databases of beacons produced with this scheme are almost half the size of the other schemes.
+//
+//nolint:dupl
+func NewPedersenBLSUnchainedG1() (cs *Scheme) {
+	var Pairing = bls.NewBLS12381Suite()
```

Comment:
Can we explicit put the domains here, even though not needed,  so it's clear the difference between this one and the previous one ? We can change later on but let's make the distinction between the two clear .

Reply:
That's making all of our Point structs a bit fatter (a `nil` DST means "use the default", whereas specifying it would carry it around in all our Point structs, so I'd prefer not to. But we're speaking about ~15 MB top given the current size of our network, I guess, so I can yeah).

I guess I could add a check in the NewBLS12381SuiteWithDST in Kyber-bls to use nil instead when the default is provided to it. I could do that in a v0.3.1 later too.

Reply:
How did you come up with 15MB?

Reply:
I've forgot tbh. 
But each signature is 1 point with 2*43 extraneous bytes, so during DKG and gossipping it can be quite a few messages, then you add up the partial beacon cache of 100 and use the in-memory DB and you'd reach at most a few MBs.